api/Crypto.json

```
@@ -94,7 +91,8 @@
               "version_added": "11"
             },
             "nodejs": {
-              "version_added": "17.4.0"
+              "version_added": "15.0.0",
+              "notes": "Since v17.4.0, a similar method is available via the <code>crypto</code> module."
```

This doesn't look right: Since the parent feature is only available via the `crypto` module from 15.0.0, and only outside of the `crypto` module from 19.0.0, isn't this note obsolete, and the current support statement (`17.4.0`) is correct?

```suggestion
              "version_added": "17.4.0"
```